### PR TITLE
Update Kava to Swift 3 and iOS 10 project settings

### PR DESCRIPTION
**Pull request by rcgilbert**
I just the Xcode 10 conversion tool for Swift 3 to make this library work properly with Xcode 8.  I also updated it to the latest project settings.

### CODE REVIEW

File: Kava/Kava/Source/PageObjects/Infrastructure/TestSession.swift [hunk 1]
```diff
@@ -13,11 +13,11 @@ import XCTest
...
  without a running TestSession.
 */
-public class TestSession {
+open class TestSession {
     
-    public private(set) var application: XCUIApplication;
+    open fileprivate(set) var application: XCUIApplication;
     
-    public private(set) var currentTest: XCTestCase
+    open fileprivate(set) var currentTest: XCTestCase
```
Comment (johnhammerlund) [inline Kava/Kava/Source/PageObjects/Infrastructure/TestSession.swift]: Both of these can be changed to `open private`

File: Kava/Kava/Source/PageObjects/Infrastructure/Block.swift [hunk 2]
```diff
@@ -113,11 +113,11 @@ public final class ApplicationBlock : Block {
 /**
  A concrete block that represents a "page", usually correlating to a view controller
 */
-public class PageBlock : Block {
+open class PageBlock : Block {
     
-    public private(set) var session: TestSession
-    public var backingElement: XCUIElement = XCUIApplication()
-    public var parentBlock: Block?
+    open fileprivate(set) var session: TestSession
```
Comment (johnhammerlund) [inline Kava/Kava/Source/PageObjects/Infrastructure/Block.swift]: Change back to `public private(set)`. Shouldn't be overridden, and it shouldn't be modified externally

File: Kava/Kava/Source/PageObjects/Infrastructure/Elements/Element.swift [hunk 3]
```diff
@@ -25,16 +25,16 @@ public protocol UITestEntity {
...
-    public let backingElement: XCUIElement
+    open let backingElement: XCUIElement
     
     // The current test session. XCUIElements cannot exist without a running XCTest.
-    public let session: TestSession
+    open let session: TestSession
     
     // The block in which the element lives, which could simply be an ApplicationBlock
-    public let parentBlock: Block
+    open let parentBlock: Block
```
Comment (johnhammerlund) [inline Kava/Kava/Source/PageObjects/Infrastructure/Elements/Element.swift]: Change all back to `public`

**Comment by johnhammerlund** [comment]
Make suggested changes and I'll merge it in. Nice work dude

**Comment by rcgilbert** [comment]
@Hammertime38 Thanks for reviewing this so fast! I added those changes you requested.  Let me know if you notice anything else that needs to be changed.

**Pull request closed without merging**